python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -59,7 +59,7 @@ async def complete_chat_async(
         """
         request_settings.messages = messages
         request_settings.stream = False
-        if request_settings.ai_model_id is None:
+        if request_settings.ai_model_id == '':
```

Comment:
why not use `if not request_settings.ai_model_id:`, with that both an empty string and a None value get handled.